extplane-server/datarefs/dataref.h

```diff
@@ -18,6 +18,10 @@ enum {
     extplaneRefTypeData = 32
 };
 
+typedef struct {
+    int lower;
+    int upper;
+} indexPair;
```

Comment:
I think you could just use a pair here:

`using indexPair = std::pair<int>;`

Then just use first and second instead of lower and upper.